sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -1160,7 +1160,6 @@ primaryExpression
     | primaryExpression DOUBLE_COLON dataType                                                  #castByColon
     | STRUCT LEFT_PAREN (argument+=namedExpression (COMMA argument+=namedExpression)*)? RIGHT_PAREN #struct
     | FIRST LEFT_PAREN expression (IGNORE NULLS)? RIGHT_PAREN                                  #first
-    | ANY_VALUE LEFT_PAREN expression (IGNORE NULLS)? RIGHT_PAREN                              #any_value
     | LAST LEFT_PAREN expression (IGNORE NULLS)? RIGHT_PAREN                                   #last
```

Comment:
How about FIRST and LAST? Seems the same case?

Reply:
Let me verify them.

Reply:
I got it. The `ANY_VALUE` missing test cases, so remove not causes any error.
`FIRST(col IGNORE NULLS)` and `LAST(col IGNORE NULLS)` already have test cases.
I will create another PR to add test cases for `ANY_VALUE(col IGNORE NULLS)`.